core/lib/di/data_di.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:core/di/app_di.dart';
+import 'package:data/providers/firebase_provider.dart';
+import 'package:data/repositories/dish_repository_impl.dart';
+import 'package:domain/domain.dart';
+import 'package:domain/usecases/fetch_all_dishes.dart';
+
+final DataDI dataDI= DataDI();
+
+class DataDI{
```

Comment:
Replace it to data layer

Reply:
and put DI folder to lib

Reply:
i thought it should be in core because i saw it in the project you gave us